prefab/curl-ssl/README.md

```diff
@@ -1,34 +1,38 @@
-# curl-ssl
+curl-ssl
+========
 
-The curl-ssl sample shows how to import native dependencies from Maven. This
-sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
-recent 10 reviews submitted to AOSP's code review system.
+This sample shows how to import [curl] and [OpenSSL] C/C++ dependencies from Google Maven
```

Comment:
This sample shows how to import [curl] and [OpenSSL] from Google Maven...

Reply:
done